includes/class-wc-stripe-api.php

```diff
@@ -632,4 +653,18 @@ private static function get_stripe_request_id( $response ) {
 		}
 		return '';
 	}
+
+	/**
+	 * Get the masked secret key.
+	 * It uses the same pattern as the Stripe dashboard: sk_live_...JLWaeq.
+	 *
+	 * @return string The masked secret key.
+	 */
+	private static function get_masked_secret_key(): string {
+		$key = self::get_secret_key();
+		if ( empty( $key ) ) {
+			return 'secret_key_not_configured';
+		}
+		return substr( $key, 0, 8 ) . '...' . substr( $key, -6 );
```

Comment:
Minor non-blocking thought: might it be worth using `***` rather than `...`, as that's more generally used as a way to represent masked or hidden data? If we want to be _really_ fancy we could use `•••` too, but I think it may be more reliable to stick with an ASCII value.
```suggestion
		return substr( $key, 0, 8 ) . '***' . substr( $key, -6 );
```

Reply:
I used `...` to be consistent with how Stripe displays the masked keys in their Logs dashboard, but I don't have a strong opinion about the character used 🤷🏼.